assets/javascripts/discourse/templates/modal/bulk-assign.hbs

```diff
@@ -0,0 +1,20 @@
+<div>
+  <AssignUserForm
+    @model={{this.model}}
+    @closeModal={{this.closeModal}}
```

Comment:
Is this used? I don't see any reference to `closeModal` in the `AssignUserForm` implementation 🤔